### PR TITLE
Update website deployment

### PR DESCRIPTION
**Pull request by ashokirla**

### CODE REVIEW

**Comment by ifeilder** [comment]
@ashokirla Does this PR handle all the homepage test cases

File: .github/workflows/ABC.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ jobs:
   build:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
+    environment: Test
 
     # Steps represent a sequence of tasks that will be executed as part of the job
```
Comment (JannesPeters) [inline .github/workflows/ABC.yml:24]: ASDAS

File: .github/workflows/ABC.yml [hunk 2]
```diff
@@ -19,6 +19,7 @@ jobs:
   build:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
+    environment: Test
```
Comment (JannesPeters) [inline .github/workflows/ABC.yml:22]: ASDAS

File: .github/workflows/ABC.yml [hunk 3]
```diff
@@ -19,6 +19,7 @@ jobs:
   build:
```
Comment (JannesPeters) [inline .github/workflows/ABC.yml:19]: ASDASDASD

**Review by JannesPeters — CHANGES_REQUESTED** [review]

**Comment by ashokirla** [comment]
New comment1234

**Comment by ashokirla** [comment]
Comment12345

**Pull request closed without merging**